cmd/abc/appbase_import.go

```diff
@@ -334,3 +339,17 @@ func verifyConnections(adaptors map[string]adaptor.Adaptor) error {
 	}
 	return nil
 }
+
+func verifyConnectionsWithoutDestination(srcConfig map[string]interface{}) error {
+	var config = make(map[string]interface{})
+	for k, v := range srcConfig {
+		config[k] = v
+	}
+	ad, _ := adaptor.GetAdaptor(srcConfig["_name_"].(string), config)
```

Comment:
Skipping the error check here would cause a failure, if we don't support the source or the source type is misspelt.

Reply:
Are you suggesting appname and File path check? I have made a commit for it. Please check.

Reply:
I meant turning that line `ad, _ := ` into `ad, err := `. And then adding an `if err != nil` check right below.

`adaptor.GetAdaptor` would return an error if you pass it a wrong name.

Reply:
Done👍🏻